src/js/video.js

```diff
@@ -484,6 +453,29 @@ videojs.off = Events.off;
  */
 videojs.trigger = Events.trigger;
 
+/**
+ * A cross-browser XMLHttpRequest wrapper. Here's a simple example:
+ *
+ *     videojs.xhr({
+ *       body: someJSONString,
+ *       uri: "/foo",
+ *       headers: {
+ *         "Content-Type": "application/json"
+ *       }
+ *     }, function (err, resp, body) {
+ *       // check resp.statusCode
+ *     });
+ *
+ * Check out the [full
+ * documentation](https://github.com/Raynos/xhr/blob/master/README.md)
```

Comment:
https://github.com/Raynos/xhr/blob/v2.1.0/README.md would be safer but not a huge issue.